src/components/models/Block1IntroTemplate.js

```diff
@@ -70,14 +86,15 @@ const Block1IntroTemplate = ({ title, markImage: { publicURL: markImageUrl }, is
               key={itemTitle}
               className={itemContainer}
               container
-              direction={index % 2 ? 'row-reverse' : 'row'}
+              direction={index % 2 || isWidthDown('xs', width) ? `${itemDirection}-reverse` : itemDirection}
```

Comment:
This seems unnecessary and fairly confusing logic to have multiple calls to `isWidthDown` and the `||` condition with two completely unrelated things.

Let's just do something like:
```javascript
<Grid direction={isWidthDown('xs', width) ? 'column-reverse' : `row${index % 2 ? '-reverse' : ''}`
```

Reply:
Much easier 💃